packages/angular-ivy/README.md

```diff
@@ -16,7 +16,7 @@
 
 ## Angular Version Compatibility
 
-This SDK officially supports Angular 12-15 with Angular's new rendering engine, Ivy.
+This SDK officially supports Angular 12 and newer with Angular's new rendering engine, Ivy.
```

Comment:
```suggestion
This SDK officially supports Angular 12 to 16 with Angular's new rendering engine, Ivy.
```

shouldn't we still be explicit here, maybe? For the future 😅 Also I guess we still need to bump peer deps, right? Or are we doing this separately?

Reply:
Yeah good point, you're right. Let's stay explicit.